app/components/apply_applications/trainee_data/view.html.erb

```diff
@@ -20,6 +20,14 @@
           <% end %>
         <% end %>
 
+        <%= render InformationSummary::View.new(renderable: @form.errors.empty?) do |component| %>
```

Comment:
FYI the logic behind when this should render may need to be abstracted further as part of this ticket https://trello.com/c/S2tXUwX1/2219-l-apply-draft-confirm-page-invalid-data-validation. We might need one high order component to switch between the `ErrorSummary::View` and the `InformationSummary::View` depending on the state of the form